app/src/lib/git/diff.ts

```diff
@@ -267,9 +267,9 @@ export async function convertDiff(
   commitish: string,
   lineEndingsChange?: LineEndingsChange
 ): Promise<IDiff> {
-  if (diff.isBinary) {
-    const extension = Path.extname(file.path)
+  const extension = Path.extname(file.path).toLowerCase()
```

Comment:
This actually catches a bug where `.PNG` is skipped as a file extension, rather than `.png`. We talked about that in #3497 and I wanted to ensure this didn't get missed.